### ISSUE TITLE
PowershellRewrite: various improvements

### ISSUE THREAD

**Issue by Jimadine** [Jimadine/telepen-sentry-daily-members-update-from-exlibris-alma#2]
- Separate out code into functions
- Add standard footer to emails (script name + date), e.g.:
```
  $emailBody = @"
${Message}

This email was generated by $(Split-Path -Path $PSCommandPath -Leaf) at $((Get-Date).ToString('dd/MM/yyyy HH:mm:ss')) on $(([System.Net.Dns]::GetHostByName(($env:computerName))).Hostname).
"@
```

**Comment by Jimadine** [comment]
The second bullet was addressed in 5d7d421d3e0bbf3712410320982eec71499426ec.

**Comment by Jimadine** [comment]
The first bullet was addressed in 5dd54f2f4ca646ecc3efa5337a331f2e60ff1d4a.
